python/ray/tune/examples/pbt_transformers/pbt_transformers.py

```diff
@@ -0,0 +1,206 @@
+# flake8: noqa
+# yapf: disable
+
+import os
+import ray
+from ray.tune import CLIReporter
+from ray.tune.schedulers import PopulationBasedTraining
+
+from ray import tune
+import trainer
```

Comment:
Can you put everything into one file?

Reply:
actually this is fine - but please be sure to have absolute imports

Reply:
done